### PR TITLE
Add prefix mis_ for all attributes from attrs.xml

### PR DESCRIPTION
**Pull request by bezmian**

### CODE REVIEW

**Comment by radzio** [comment]
Please add **resourcePrefix**

File: material-intro-screen/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 <resources>
 
-    <style name="Theme.Intro" parent="Theme.AppCompat.NoActionBar">
+    <style name="mis_Theme.Intro" parent="Theme.AppCompat.NoActionBar">
         <item name="android:windowBackground">@android:color/transparent</item>
         <item name="android:windowIsTranslucent">true</item>
     </style>
 
-    <style name="ColoredButton" parent="Widget.AppCompat.Button.Colored">
+    <style name="mis_ColoredButton" parent="Widget.AppCompat.Button.Colored">
```
Comment (radzio) [inline material-intro-screen/src/main/res/values/styles.xml]: Are you sure that this is good practice?

File: material-intro-screen/src/main/res/values/styles.xml [hunk 2]
```diff
@@ -1,12 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
 
-    <style name="Theme.Intro" parent="Theme.AppCompat.NoActionBar">
+    <style name="mis_Theme.Intro" parent="Theme.AppCompat.NoActionBar">
```
Comment (radzio) [inline material-intro-screen/src/main/res/values/styles.xml]: Check next commpent.

**Review by radzio — CHANGES_REQUESTED** [review]

File: material-intro-screen/src/main/res/values/styles.xml (hunk 2 above)
Comment (bezmian) [reply material-intro-screen/src/main/res/values/styles.xml]: I need to add this as resourcePrefix from gradle forces me to do that.

File: material-intro-screen/src/main/res/values/styles.xml (hunk 1 above)
Comment (bezmian) [reply material-intro-screen/src/main/res/values/styles.xml]: I need to add this as resourcePrefix from gradle forces me to do that.

**Review by radzio — APPROVED** [review]

**Comment by radzio** [comment]
LGTM!

**Pull request merged**